code-reviews/example2/math_operations.py

```diff
@@ -0,0 +1,16 @@
+# Common mathemmatical operations
+
+def add(a, b):
+    return a + b
+
+def subtract(a, b):
+    return a - b
+
+def multiply(a, b):
+    return a * b
+
+
+def divide(a, b):
+    if a == 0:
+        raise ValueError("Cannot divide by zero")
```

Comment:
Can it be a zerodivisionerror instead of valueError

Reply:
agreed, this would be a better error to sure